modules/auxiliary/gather/zoomeye_domain.rb

```diff
@@ -0,0 +1,130 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+#
+##
+
+class MetasploitModule < Msf::Auxiliary
+    include Msf::Auxiliary::Report
+
+    def initialize(info={})
+        super(update_info(
+                info,
+                "name" => "ZoomEye Domain",
+                "Description" => %q{
+            The module use the ZoomEye API to search ZoomEye.ZoomEye is a cyberspace search engine,
+            users can search for network devices using a browser.
+            },
+                'Author' => ['wh0am1i'],
+                "References" => [
+                  ["URL", "https://www.zoomeye.org/api/doc"],
+                  ["URL", "https://github.com/knownsec/ZoomEye-python"]
+                ],
+                "License" => MSF_LICENSE
+              ))
+
+        register_options(
+          [
+            OptString.new('APIKEY', [true, 'The ZoomEye API KEY']),
+            OptString.new('QUERY', [true, 'The ZoomEye dork']),
+            OptInt.new('PAGE', [true, "Max amount of pages to collect", 1]),
```

Comment:
Using `MAXPAGE` for consistency with the shodan module would be helpful.
```suggestion
            OptInt.new('MAXPAGE', [true, "Max amount of pages to collect", 1]),
```